101-lab/content/06_application_availability.md

```diff
@@ -23,7 +23,7 @@ Adjust the settings to match the images below:
 Single pod applications are not highly-available and can be abruptly terminated if the pod crashes or due to regularly 
 scheduled platform maintenance. In order to simulate the effect on an application: 
 
-- Navigate to the pod, select `Actions` and `Delete`
+- Navigate to the pod under Administrator > Workloads > Pods, select your Rocketchat pod, select `Actions`(on the right hand side) and `Delete Pod`
```

Comment:
What's the thought behind this change? There's more than one way to navigate to the pod in the GUI, and in the video we demo from the developer view. So I'd prefer to either leave it vague, or describe it consistent with the video demo (developer>topology>deployment>pods)

Reply:
Agree, updated